libs/perun/components/src/lib/groups-list/groups-list.component.html

```diff
@@ -115,12 +113,13 @@
         </th>
         <td *matCellDef="let group" mat-cell>
           <i
-            class="material-icons {{getStatusAttribute(group) | groupStatusIconColor}}"
-            matTooltip="{{getStatusAttribute(group)}}"
+            *ngIf="{status: group.attributes | findAttribute: 'groupStatus'} as groupStatus"
+            class="material-icons {{groupStatus.status | groupStatusIconColor}}"
```

Comment:
I am not sure if the pipe will fail due to it, but there s an error: `Invalid number of arguments, expected 2..3 `

Reply:
It may cause problems in some specific circumstance. However I won't be fixing is here as it is already on master. Could be looked at in different issue.